recipes/date/all/patches/cmake.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f025a3a..072c644 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -6,9 +6,6 @@ include( GNUInstallDirs )
+ 
+ find_package( Threads REQUIRED )
+ 
+-# Override by setting on CMake command line.
+-set( CMAKE_CXX_STANDARD 17 CACHE STRING "The C++ standard whose features are requested.")
+-
+ option( USE_SYSTEM_TZ_DB "Use the operating system's timezone database" OFF )
+ option( USE_TZ_DB_IN_DOT "Save the timezone database in the current folder" OFF )
+ option( BUILD_SHARED_LIBS  "Build a shared version of library" OFF )
+@@ -61,6 +58,11 @@ else( )
+ 	set( OPTIONAL_LIBRARIES ${CURL_LIBRARIES} )
```

Comment:
Does it properly translate all libcurl definitions (`CURL_STATICLIB=1` for example if libcurl static) and transitive dependencies? If not, maybe using conan targets would help (very robust by experience).

Reply:
good point i'll take a look at this.

Reply:
Okay, this will be tricky - I didnt look at this before - just inherited it from previous PR.
It seems on windows it 
maybe should be using USE_AUTOLOAD=1 always ?
But instead what the changes are doing is they enable always libcurl on windows but disable autoload by default hmm...

Reply:
Okay, I have reworked this now.

I think its correct now.

I've tested this though - AUTO_DOWNLOAD does not seem to work anymore (possible iana website has changed ).

However it works when invoking `download_version` `install_version `